common/app/routes/Settings/Toggle-Button/Toggle-Button.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ToggleButtonGroup as BSBG, ToggleButton as TB } from 'react-bootstrap';
+
+import ns from './ns.json';
+
+const propTypes = {
+  onChange: PropTypes.func.isRequired,
+  value: PropTypes.bool.isRequired
+};
+
+export default function ToggleButton({
+  onChange,
+  value
+}) {
+  return (
+    <div className={ `${ns}-container` }>
+      <BSBG
+        name='monthly-email'
```

Comment:
Should this be handled by props if it is a generic component?

Reply:
Nice catch